src/citrine/_serialization/properties.py

```diff
@@ -498,10 +498,13 @@ def _deserialize(self, value: typing.Any) -> typing.Any:
                 return prop.deserialize(value)
             except ValueError:
                 pass
+            except RuntimeError:
+                # passes on RuntimeError from Property.deserialize_from_dict
+                pass
         # this is a failsafe that shouldn't actually ever get hit, hence no cover
         msg = "An unexpected error occurred while trying to deserialize {} to one of the " \
               "following types: {}.".format(value, self.underlying_types)  # pragma: no cover
-        raise ValueError(msg)  # pragma: no cover
+        raise RuntimeError(msg)  # pragma: no cover
```

Comment:
Why change the error type?

Reply:
I think it was incorrect. RuntimeError is consistent with the serialize method, and other similar errors thrown in this file.